### PR TITLE
Allow permalinking to lines

### PR DESCRIPTION
**Pull request by ammaraskar**
Allows you to click on lines and grab a hash based permalink to it in your url bar, much like in github.
### Demonstration: http://paste.md-5.net/pomiqeyeye.cs#L17

There is one hack in here I'm not real proud of: https://github.com/seejohnrun/haste-server/pull/43/files#L3R21 but I can't really figure out a nice way to work around that properly. Basically when you change the hash, it calls the pop event because the history changes. Feel free to offer any better solutions to the problem if you have any.

### CODE REVIEW

**Comment by gt-c** [comment]
Can I expect this to come soon?

**Comment by seejohnrun** [comment]
@gt-c Just given the age of the PR we'll need some cleanup here / shouldn't be too hard to get it up to date but it'll be a bit before I have time personally

**Comment by gt-c** [comment]
ok :D thanks!

File: static/solarized_dark.css [hunk 1]
```diff
@@ -98,3 +98,7 @@ pre .tex .formula,
 pre .code {
   background: #073642;
 }
+
+.line.highlight {
+  background-color: #004A5E;
+}
```
Comment (filipechagas) [inline static/solarized_dark.css:104]: Can you please fix EOL?

**Review by filipechagas — CHANGES_REQUESTED** [review]
Hi @ammaraskar 
Thank  you for contributing!

Can you please verify @seejohnrun comments and also one of my own?
Also, please update the branch with latest `master` changes.

**Pull request closed without merging**